Mathlib/Tactic/DocCommands.lean

```diff
@@ -11,6 +11,9 @@ import Mathlib.Lean.Expr.Basic
 -/
 open Lean Elab
 
+-- TODO we need to check whether `inherit_doc` attribute works on foreign definitions.
```

Comment:
Did you try that?

Reply:
Ah, Mario meant "foreign defs" == "defs from imported files", I'll check whether `copy_doc_string` & `inherit_doc` support defs from imported files.

Reply:
It turns out both `copy_doc_string` & `inherit_doc` can't handle definitions from other files.

`copy_doc_string hi → foreign_def`
<img width="440" alt="image" src="https://user-images.githubusercontent.com/7578559/202056649-b97fd805-3d5b-4f88-9238-b539f610ffe8.png">

`attribute [inherit_doc hi] one two foreign_def`
<img width="442" alt="image" src="https://user-images.githubusercontent.com/7578559/202057035-27d42d22-de7b-4f12-beee-1ab7b0162d0a.png">

@digama0 / @gebner, should I just remove the `copy_doc_string` implementation from Mathlib then (since it can be entirely superseded by `attribute [inherit_doc ~]`)?